app/models/dog_searcher.rb

```diff
@@ -77,14 +77,25 @@ def with_includes
   end
 
   def with_sorting
-    sort = if text_search?
-      sort_string = "case when name ilike ? then 1 else 2 end, ? ?"
-      [sort_string, search_term + '%', sort_column, sort_direction]
-    else
-      ['? ?', sort_column, sort_direction]
-    end
+    @dogs = @dogs.order(sort_column + ' ' + sort_direction)
+
+    # Falling back to the old sort stuff here for now
```

Comment:
Given that this is in git, is it useful to keep the commented out code without explanation for why it's commented out?